test/type/string_test.exs

```diff
@@ -112,4 +112,79 @@ defmodule Litmus.Type.StringTest do
                {:error, "#{field} length must be #{length} characters"}
     end
   end
+
+  describe "regex validation" do
+    test "returns :ok when value matches the regex pattern" do
+      data = %{"zip_code" => "94107-1741"}
+
+      schema = %{
+        "zip_code" => %Litmus.Type.String{
```

Comment:
I might stick with something more general, like a password. It's not _super_ important, but I think zip code pertains more to us 😃

Reply:
zip codes are cool though. i say levae it :P